submitr/submission.py

```diff
@@ -674,7 +767,11 @@ def _check_ingestion_progress(uuid, *, keypair, server) -> Tuple[bool, str, dict
     From outer scope: server, keypair, uuid (of IngestionSubmission)
     """
     tracking_url = ingestion_submission_item_url(server=server, uuid=uuid)
-    response = portal_request_get(tracking_url, auth=keypair, headers=STANDARD_HTTP_HEADERS).json()
+    response = portal_request_get(tracking_url, auth=keypair, headers=STANDARD_HTTP_HEADERS)
```

Comment:
Why not just use ``ff_utils.get_metadata``?

Reply:
Now starting use dcicutils.portal_utils.Portal wrapper which will eventually supplant this.